### DISCUSSION TITLE
SmartAutoClicker-2.2.0: Tutorials - Give me your feedback !

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#300] [Announcements]
Hi everybody !

I'm very exited to introduce the new tutorials integrated directly within the app. This has been a lot of work and required some rework of the ui engine in order to create what I had in mind. I really hope you will like it.

If you took the time to test it, can you provide me some feedback/ideas in order to improve this feature ? Maybe requests for a future tutorial on a specific feature!

**Comment by c-sanchez** [comment]
Please add some tutorial for the Intents, I think it is one of the most complex functions and maybe the most useful too, however an average user will not know how to use any of its advanced options.

---

I want to use intent to switch between two recent applications, however the default intent when opening another application does it from the start and not returning to its previous state, as when opening it from the recent apps menu.

> Type: Start Activity
> Action: android.intent.action.MAIN
> Flags: 335544320
> Component name: http://com.binance.dev/com.eaas.launcher.activities.main.MainActivity

What should I change in the options to make it open the application from the recent menu and not from the start?

**Comment by CreativeFlug** [comment]
Tutorial on intents would be nice. 
I would really like to send an intent to Tasker, so it can do stuff as well as the auto clicker. Have tried to figure out how, but I can't figure it out.

**Comment by Nain57** [comment]
I'm actually planning on removing the advanced intent feature.

It was a feature intended for advanced users (i.e developers), and no matter the way I explain that it's documented on Android Developers and many tutorials are available over the internet, people keep coming at me asking for explaining or new features on it. 

And when I say keep coming, it's literally harassment at this point. By mail, on the PlayStore, on Reddit...

The API provided by the app reflect the whole Android API, the content of each Intent are determined by the developers of others apps, there is nothing more to explain 

This feature has been more hurting to the application than it is beneficial, so it will be removed in the next major release

**Reply by CreativeFlug** [reply]
That's sad to hear, what I became confused by, was that in all the tutorials I saw, neither the flag or the component was necessary, at least that is how I understood it, while in the smart auto clicker app, both are required. 
But since there are so many people asking questions about it and making a fuzz about it, it's probably best to remove it all together as you said. Thanks for the reply though!

**Reply by Nain57** [reply]
Thanks for understanding.

I actually started receiving threats by mail, mass indian people asking for a refund and saying that if I don't respond within 3 days, I quote "I will regret it". Alongside with people requesting WTF features and getting mad and insulting you because you won't implement it.
Oh, and people (mainly India again) rating 1* with no logical explanation (even sometimes a positive coments)

Due to this toxic environment, I will step back from the development of Smart Autoclicker, developers are not slaves of their customers.

If this behavior from people keeps impacting my daily mood so much, I might close this repository and unpublish the application.

Sorry for the abrupt announcement, see you all in a few month.

**Reply by zacharyblasczyk** [reply]
This is awful and I am sad to hear you are harassed and will be removing it.

Literally endless possibilities with this, but you have to be a dev and read the docs to use.

Maybe a compromise is to build one version for devs and start a forum where people can contribute their own walkthroughs, and the play version which can axe it.

I would ask that if you remove this feature completely, you would add an ability to chime or vibrate if an image is detected on screen into the app.

This is super useful when I want to be notified of something, but take manual action.
